app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -60,6 +80,9 @@ const mapDispatchToProps = dispatch => ({
 SearchAndAddOrder.propTypes = {
   fetchInpatientCareSetting: PropTypes.func.isRequired,
   fetchOutpatientCareSetting: PropTypes.func.isRequired,
+  drug: PropTypes.object,
```

Comment:
PropTypes.object is deprecated, you may consider checking out PropTypes.shape({}) instead